README.md

```diff
@@ -1,70 +1,73 @@
-# Getting Started with Create React App
+# Fruitinfo
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+Fruitinfo is integrated with Fruityvice API. Fruitinfo is provides data for all kinds of fruit! Users can use Fruitinfo to find out interesting information about fruit and educate yourself.
 
-## Available Scripts
+## Built With
 
-In the project directory, you can run:
+- React 
+- Redux
+- Sass
+- React-bootstrap 
+- JSX
 
-### `npm start`
+## Live Demo
+#comming sooon
 
```

Comment:
- [ ] One of the requirement is to deploy the project, I have read your comment on the description, however I strongly suggest you deploy it as its a requirement and after you change the default branch most of the deploying platforms redeploy you site or you can change which branch should be deployed on netlify or heroku settings.